fiftyone/plugins/utils.py

```diff
@@ -219,3 +222,103 @@ def _do_get_plugin_info(task):
             spec = gh_repo
 
         logger.debug("Failed to retrieve plugin info for %s: %s", spec, e)
+
+
+def get_target_view(ctx, target):
```

Comment:
Instead of this using a string as a param I would prefer it using the enum property you referred to elsewhere.

Reply:
In the [`@voxel51/utils`](https://github.com/voxel51/fiftyone-plugins/blob/6d1de93dee2cd32d04b1f3071bbbb844697c00c5/plugins/utils/__init__.py#L1708) plugin it also made sense for me to include `SELECTED_LABELS`:

```py
def _get_target_view(ctx, target):
    if target == "SELECTED_LABELS":
        return ctx.view.select_labels(labels=ctx.selected_labels)

    if target == "SELECTED_SAMPLES":
        return ctx.view.select(ctx.selected)

    if target == "DATASET":
        return ctx.dataset

    return ctx.view
```

The way I see it, there's two common patterns:
1. The plugin only cares about media, in which case the only relevant choices are `DATASET/VIEW/SELECTED_SAMPLES`
2. The plugin cares about labels, in which case the relevant choices also include `SELECTED_LABELS`

So IDK what the best way to build a reusable utility is that covers both cases.